tests/testcases/mod.rs

```diff
@@ -734,6 +734,38 @@ pub async fn simple_spin_rust_works(controller: &dyn Controller) {
     tc.run(controller).await.unwrap()
 }
 
+pub async fn simple_redis_rust_works(controller: &dyn Controller) {
+    async fn checks(
+        _: AppMetadata,
+        _: Option<Pin<Box<dyn AsyncBufRead>>>,
+        stderr_stream: Option<Pin<Box<dyn AsyncBufRead>>>,
+    ) -> Result<()> {
+        let stderr = utils::get_output_stream(stderr_stream, Duration::from_secs(5)).await?;
```

Comment:
Our other Redis Rust test uses a 20 second timeout.  We should maybe pick a good timeout and give it a const name.

Reply:
We always use 5 second for getting the output stream. The 20 seconds is to wait for spin to become ready. I can try to make that clearer, but I don't want to go too beyond what this PR is for.

Reply:
Nope, no worries, apologies for missing the different contexts.